### PR TITLE
Redstone 2 Chart Fix - chart clip fix

### PR DESCRIPTION
**Pull request by vvgeorgiev**

### CODE REVIEW

**Comment by Hujerek** [comment]
It looks the fix is working fine. Thanks.
But still there is an issue with missing (or wrong color) grid lines (dashed X grid-lines) !
See https://github.com/telerik/UI-For-UWP/issues/66

File: Controls/Chart/Chart.UWP/Visualization/CartesianChart/Series/CartesianSeries.cs [hunk 1]
```diff
@@ -106,6 +112,52 @@ internal void ChartAxisChanged(CartesianAxis oldAxis, CartesianAxis newAxis)
...
+        {
+            if (!this.ClipToPlotArea || !this.IsRendered || !RadChartBase.IsRedstone2())
+            {
+                return;
+            }
+
+            foreach (var path in this.renderSurface.Children.OfType<Path>())
+            {
+                Point topLeft;
+                if (path.Data.Bounds != new Rect())
```
Comment (TKonov) [inline Controls/Chart/Chart.UWP/Visualization/CartesianChart/Series/CartesianSeries.cs]: Use static filed instead or check rect dimensions

File: Controls/Chart/Chart.UWP/Visualization/CartesianChart/Series/CartesianSeries.cs [hunk 2]
```diff
@@ -106,6 +112,52 @@ internal void ChartAxisChanged(CartesianAxis oldAxis, CartesianAxis newAxis)
...
+                }
+                else
+                {
+                    topLeft = path.GetTopLeft();
+                    topLeft.X = topLeft.X - this.GetStrokeThicknessAdjustment();
+                    topLeft.Y = topLeft.Y - this.GetStrokeThicknessAdjustment();
+                }
+
+                TransformGroup transform = path.RenderTransform as TransformGroup;
+                Point translate = new Point();
```
Comment (TKonov) [inline Controls/Chart/Chart.UWP/Visualization/CartesianChart/Series/CartesianSeries.cs]: Think of a better name of point variable to avoid confusion

**Comment by Hujerek** [comment]
Are you actually working on the issue with missing (or wrong color/theme) grid lines (dashed X grid-lines) ?

**Pull request merged**